client/src/pages/NewTree/NewTreePlanter.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { FormTextField } from '@/components/Form';
-import Section from '../Section';
+import Section from '@/components/Section/Section';
```

Comment:
GitHub doesn't allow me to comment on empty files so posting here. [Section/Section.scss](https://github.com/waterthetrees/wtt_front/blob/feature/newstructure-newvectortiles/client/src/components/Section/Section.scss) is empty so can we remove it completely?